_docs/using/intents.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Intents
+category: Using the bot
+order: 8
+---
+
+#### How to enable Intents?
+1. Go to Discord's [applications page](https://discord.com/developers/applications/me) (logging in if prompted).
+2. Select your bot.
+3. Go to the page called "Bot", and scroll until you see "Privileged Gateway Intents"
+4. Now enable "PRESENCE INTENT" and "SERVER MEMBERS INTENT" and you should be good to go!
```

Comment:
Shouldn't we have them enable message content as well since we use on_message as a command handler?

Reply:
Good one, I'll add it aswell just to be safe